package.json

```diff
@@ -1,8 +1,13 @@
 {
   "name": "svelte-i18n",
   "version": "3.7.4",
-  "main": "dist/runtime.cjs.js",
-  "module": "dist/runtime.esm.js",
+  "type": "module",
+  "main": "dist/runtime.mjs",
```

Comment:
There's little harm in using the mjs extension here, but using the standard js extension would be equivalent and might be preferable (e.g. I've seen some CDNs not know how to handle the Node extensions which I remember creating an issue on the REPL)

Reply:
> - use of .mjs and .cjs extension instead of .esm.js and .cjs.js (doing such change make my both Svelte3/ts-nodeCjs and Svelte4/ts-nodeEsm projects unit tests and build working properly, better compared with my https://github.com/kaisermann/svelte-i18n/issues/219 issue comment).

Actually using just the .cjs extension is enough for my two projects to work properly . I do not have a strong opinion on using the .mjs extension and anyway it is not required as `"type": "module",` is present in package.json.

I will change that.